src/java.base/share/classes/sun/security/provider/certpath/BuildStep.java

```diff
@@ -187,27 +187,25 @@ public int getResult() {
      * @return String string representing meaning of the result code
      */
     public String resultToString(int res) {
-        String resultString = "";
+        String resultString;
         switch (res) {
-            case POSSIBLE:
-                resultString = "Certificate to be tried.\n";
-                break;
-            case BACK:
-                resultString = "Certificate backed out since path does not "
-                    + "satisfy build requirements.\n";
-                break;
-            case FOLLOW:
-                resultString = "Certificate satisfies conditions.\n";
-                break;
-            case FAIL:
-                resultString = "Certificate backed out since path does not "
-                    + "satisfy conditions.\n";
-                break;
-            case SUCCEED:
-                resultString = "Certificate satisfies conditions.\n";
-                break;
-            default:
-                resultString = "Internal error: Invalid step result value.\n";
+        case POSSIBLE:
```

Comment:
Brad pointed me to this:
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

I think your reference is better since we're working on OpenJDK.

Reply:
Never mind. I see what you're suggesting.